src/openms/source/FORMAT/HANDLERS/MzIdentMLDOMHandler.cpp

```diff
@@ -990,18 +990,14 @@ namespace OpenMS
                         const ResidueModification* m = ModificationsDB::getInstance()->getModification(mname, r, ResidueModification::C_TERM);
                         mod = m->getFullId();
                       }
-                      else if (spci->second.front().getAccession() == "MS:1002057")  // pro nterm
+                      else if (spci->second.front().getAccession() == "MS:1002057")  // protein nterm
```

Comment:
maybe change nterm-> N-term

Reply:
these comments were just part of the session where I showed how to comment on PR for beginner. Feel free to ignore